packages/core/src/utils/nodes.js

```diff
@@ -201,15 +201,15 @@ export const isScopeCreator = (node: Node) =>
     EXPRESSIONS_TYPES.ARROW_FUNCTION_EXPRESSION
   ].includes(node.type);
 
-export const isFunction = (node: Node) =>
-  [
+export const isFunction = (node: ?Node) =>
```

Comment:
Oh, In which cases we can provide `undefined` to `isFunction`?
I think that it's the invalid case when we provide `undefined` as an argument for the function :(

Reply:
it was done to simplify code in getBody
```
...ensureArray(currentNode.arguments).filter(a => !NODE.isFunction(a)),
```
in this line if arguments are empty it will result in array with 1 undefined
so this check in isFunction should prevent it
but I can rework  this to make isFunction always work with values